packages/big-design/src/components/Table/Table.tsx

```diff
@@ -95,10 +96,30 @@ const InternalTable = <T extends TableItem>(props: TableProps<T>): React.ReactEl
       if (typeof onRowDrop === 'function') {
         onRowDrop(source.index, destination.index);
       }
+
+      setHeaderCellWidths([]);
     },
     [onRowDrop],
   );
 
+  const onBeforeDragStart = () => {
+    const headerCellIconId = window.document.getElementById('header-cell-icon');
```

Comment:
🍹 Maybe keeping track of a `ref` for this `th` may be better than having an `id` 🤔

Reply:
@chanceaclark you're right 👍🏼 . For getting the `width`, I used `offseetWidth` that apparently considers the padding and boarding as well.  Reference: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/offsetWidth. 
Is that ok?

Reply:
Should should be :gucci: